CMSgov/onemac
# OY2 9314 Update the Role Name from State User to State Submitter

Story: https://qmacbis.atlassian.net/browse/OY2-9314
EndPoint: https://d24ln22m3h884q.cloudfront.net

### Details
There is confusion around the use of State User to designate both the IDM role (for submitters and admins) and the submitter role.  The decision was to strictly identify the Portal Role as the State Submitter and update all references to the “State User” accordingly.

NOTE: This PR does NOT include the scripts that will be needed for the data migration on VAL (PROD should not need).  I feel like that was best done as a separate branch.

### Changes
- Updated confluence documentation
- Changed stateuser to statesubmitter in code and test/seed data
- Updated labels for use in tables
- Updated email content and subject headings
- Updated Nightwatch tests
- Updated environment variables and github secrets
- Added pauses to Nightwatch tests for window opening issues

### Implementation Notes
GitHub makes it easy to see what I’ve changed… and not what I’ve missed… Please use whatever tricks you have to discover areas I may have missed.  Yes, throw those pies at my face!

### Test Plan:
1. Verify Nightwatch and Unit tests ran and are green
2. Open endpoint as a helpdesk user and verify the State User is marked as State Submitter.
3. Try to find anyplace I missed updating the role
4. Verify statesubmitter* development logins work and stateuser* do not
